fem/fespace.cpp

```diff
@@ -2800,7 +2800,8 @@ ElementRestriction::ElementRestriction(const FiniteElementSpace &f,
    {
       for (int d = 0; d < dof; ++d)
       {
-         const int gid = elementMap[dof*e + d];
+	 const int sgid = elementMap[dof*e + d];  // signed
```

Comment:
Can you point me to where the signed behavior of the `elementMap` is documented?

Reply:
I don't think it is documented anywhere, but signed DOF's are heavily used in mfem. Searching for "-1-" in fespace.* produces many results, for example in FiniteElementSpace::GetElementDofs. Negative indices indicate negative DOF orientation, used in H(curl) and H(div).